docker-multinode/common.sh

```diff
@@ -45,6 +45,8 @@ kube::multinode::main(){
     -v /var/run:/var/run:rw \
     -v /var/lib/docker:/var/lib/docker:rw \
     -v /var/lib/kubelet:/var/lib/kubelet:shared \
+    -v /etc/cni/net.d:/etc/cni/net.d:rw \
```

Comment:
I don't see any reason to keep these lines when we're packaging things in the container